bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/dhcp/DHCPListenService.java

```diff
@@ -34,7 +34,7 @@
 @NonNullByDefault
 public class DHCPListenService {
     static @Nullable DHCPPacketListenerServer instance;
-    private static Map<String, IPRequestReceivedCallback> registeredListeners = new TreeMap<>();
+    private static final Map<String, IPRequestReceivedCallback> registeredListeners = new TreeMap<>();
```

Comment:
Since now `final` SAT rules require renaming:
```suggestion
    private static final Map<String, IPRequestReceivedCallback> REGISTERED_LISTENERS = new TreeMap<>();
```